### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by peterrum**
... by removing the file name from the footer of some files

### CODE REVIEW

**Review by drwells — APPROVED** [review]
That explains why these were sometimes indented and sometimes not - thanks!

File: include/deal.II/opencascade/utilities.h [hunk 1]
```diff
@@ -467,7 +467,6 @@ namespace OpenCASCADE
 
 DEAL_II_NAMESPACE_CLOSE
 
-#  endif // DEAL_II_WITH_OPENCASCADE
+#endif // DEAL_II_WITH_OPENCASCADE
 
 #endif // dealii_occ_utilities_h
```
Comment (marcfehling) [inline include/deal.II/opencascade/utilities.h:472]:
```suggestion
#endif
```

Pedantic me found one last filename here, but this one obviously does not affect the indentation. You got rid of these filenames in other occasions.

**Review by marcfehling — APPROVED** [review]
Looks good!

File: include/deal.II/opencascade/utilities.h (hunk 1 above)
Comment (peterrum) [reply include/deal.II/opencascade/utilities.h]: I have removed that. I was looking for comments with file names `grep -r -e "/\*-----------" | grep ".h -". The problem is the format of the comment.

**Review by marcfehling — APPROVED** [review]

**Review by tamiko — APPROVED** [review]

**Review by tjhei — APPROVED** [review]
Woah! That is the reason for indenting the # macros?!

Thank you for fixing this.

File: include/deal.II/lac/petsc_matrix_free.h [hunk 2]
```diff
@@ -14,15 +14,15 @@
...
+#include <deal.II/base/config.h>
 
-#  ifdef DEAL_II_WITH_PETSC
-#    include <deal.II/lac/exceptions.h>
-#    include <deal.II/lac/petsc_matrix_base.h>
-#    include <deal.II/lac/petsc_vector.h>
+#ifdef DEAL_II_WITH_PETSC
+#  include <deal.II/lac/exceptions.h>
+#  include <deal.II/lac/petsc_matrix_base.h>
+#  include <deal.II/lac/petsc_vector.h>
```
Comment (drwells) [inline include/deal.II/lac/petsc_matrix_free.h:25]: I'd still like to get rid of this kind of indentation but that's outside the scope of this PR.

Comment (peterrum) [reply include/deal.II/lac/petsc_matrix_free.h:25]: That would be nice!

**Pull request merged**